mlir/include/mlir/Dialect/Rock/utility/builderUtils.h

```diff
@@ -9,6 +9,13 @@
 
 namespace mlir {
 namespace rock {
+// Utility to create an APInt of the requested type
+std::pair<APInt, bool> createAPInt(Type elemType, int32_t value);
```

Comment:
If it is APint, it is probably better to use `int64_t`. `int32_t` should be fine for the inference. but i am not sure if we will see any `uint32_t` which can't be represented in `int32_t` range.